analyses/cell-type-neuroblastoma-04/run-analysis.sh

```diff
@@ -51,32 +75,58 @@ download_file() {
   fi
 }
 
-nbatlas_subset_url="https://data.mendeley.com/public-files/datasets/yhcf6787yp/files/0a569381-3a0c-4eec-863a-e20544b686ed/file_downloaded"
+# Define URLs; the first 2 will always be downloaded
 nbatlas_full_url="https://data.mendeley.com/public-files/datasets/yhcf6787yp/files/f5969395-5f6e-4c5d-a61a-5894773d0fee/file_downloaded"
 nbatlas_tumor_url="https://data.mendeley.com/public-files/datasets/yhcf6787yp/files/78cad1b4-7425-4073-ba09-362ef73c9ab9/file_downloaded"
+nbatlas_subset_url="https://data.mendeley.com/public-files/datasets/yhcf6787yp/files/0a569381-3a0c-4eec-863a-e20544b686ed/file_downloaded"
+
+# Define NBAtlas reference object files with the same name as on Mendeley: https://data.mendeley.com/datasets/yhcf6787yp/3
+nbatlas_full_seurat="${scratch_dir}/seuratObj_NBAtlas_share_v20241203.rds" # this is needed regardless of which nbatlas_version was specified since we need to subset ids
+nbatlas_tumor_metadata_file="${scratch_dir}/SeuratMeta_Share_TumorZoom_NBAtlas_v20250228.rds"
```

Comment:
I think you could still define the paths to the full and subset together here and then just download the full or the subset depending on what version is being used.

Reply:
Ah I see in your note now that you need to have the full to get the IDs.